### PR TITLE
Gwh apidocs 1991 09 minor text edits

### PR DESCRIPTION
**Pull request by genwhittTTD**
Several mods based on suggestions from JN and others.

### CODE REVIEW

File: docs/getting-started/gs-normalization-encoding.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ When you're taking user information such as an email address, and following the
...
+- If you miss any of the required steps&#8212;for example, you hash without first normalizing&#8212;the result will not be the correct valid UID2 value for the input data.<br/>For example, let's say a data provider wants to generate a UID2 from `Jane.Saoirse@gmail.com`. This normalizes to `janesaoirse@gmail.com`, and the hashed and Base64-encoded value is `ku4mBX7Z3qJTXWyLFB1INzkyR2WZGW4ANSJUiW21iI8=`.<br/>The publisher, with the same email address, does not normalize. The hashed and Base64-encoded value for the un-normalized email, `Jane.Saoirse@gmail.com`, is `f8upG1hJazYKK8aEtAMq3j7loeAf5aA4lSq6qYOBR/w=`. These two different values result in two different UID2s. The first, processed correctly, will match other instances generated from the same original data. The second, incorrectly processed, will not.<br/>In this scenario, because the UID2 does not match other instances for the same user, the publisher misses the opportunity to benefit from targeted advertising.
```
Comment (jon8787) [inline docs/getting-started/gs-normalization-encoding.md:28]: The sentence "The publisher, with the same email address, does not normalize" confused me - I had to reread the paragraph to understand what we were getting at. Maybe we can include an adjective here like mistakenly/incorrectly.

**Review by jon8787 — APPROVED** [review]

File: docs/getting-started/gs-normalization-encoding.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/getting-started/gs-normalization-encoding.md:28]: Done thx.
The publisher, with the same email address, by mistake does not normalize.

File: docs/getting-started/gs-normalization-encoding.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ When you're taking user information such as an email address, and following the
...
+- If you miss any of the required steps&#8212;for example, you hash without first normalizing&#8212;the result will not be the correct valid UID2 value for the input data.<br/>For example, let's say a data provider wants to generate a UID2 from `Jane.Saoirse@gmail.com`. This normalizes to `janesaoirse@gmail.com`, and the hashed and Base64-encoded value is `ku4mBX7Z3qJTXWyLFB1INzkyR2WZGW4ANSJUiW21iI8=`.<br/>The publisher, with the same email address, by mistake does not normalize. The hashed and Base64-encoded value for the un-normalized email, `Jane.Saoirse@gmail.com`, is `f8upG1hJazYKK8aEtAMq3j7loeAf5aA4lSq6qYOBR/w=`. These two different values result in two different UID2s. The first, processed correctly, will match other instances generated from the same original data. The second, incorrectly processed, will not.<br/>In this scenario, because the UID2 does not match other instances for the same user, the publisher misses the opportunity to benefit from targeted advertising.
```
Comment (sunnywu) [inline docs/getting-started/gs-normalization-encoding.md:28]: Shall we not use the gmail dot-skipping rule here? I'm just concerned that if people just read this and they assume dot-skipping applies to all emails. i think using examples to show needing to normalise by converting upper to lower case is sufficient as brief example here and people can read in details further down the page to know the actual rules?

Comment (genwhittTTD) [reply docs/getting-started/gs-normalization-encoding.md]: @sunnywu: Updated. Please check? Thx!

Comment (sunnywu) [reply docs/getting-started/gs-normalization-encoding.md]: Looks good to me thx!

**Pull request merged**